### PR TITLE
Display avatar placeholder for new user.

### PR DESCRIPTION
**Pull request by kgrruz**
Allow display avatar placeholder using random string(on user creation or similar situations).

### CODE REVIEW

File: app/Entities/User.php [hunk 1]
```diff
@@ -13,14 +13,20 @@ class User extends ShieldUser
      *
      * @return string
      */
-    public function renderAvatar(int $size=52)
+    public function renderAvatar(int $size=52, $random = false)
     {
+        // Generate random string for avatar placeholder
```
Comment (lonnieezell) [inline app/Entities/User.php]: Good catch - but I wouldn't force a random parameter here. Instead, if $idString is empty after checking for names, username, email, then give it a random string then. That way it's automatic. And a bit cleaner.

File: bonfire/Modules/Users/Views/form.php [hunk 2]
```diff
@@ -32,7 +32,7 @@
                         <!-- Avatar preview and edit links -->
                         <div class="row">
                             <div class="col-12 d-flex justify-content-center">
-                                <?= $user->renderAvatar(140) ?? (new \App\Entities\User())->renderAvatar(140) ?>
+                                <?= isset($user) ? $user->renderAvatar(140) : (new \App\Entities\User())->renderAvatar(140,true) ?>
```
Comment (lonnieezell) [inline bonfire/Modules/Users/Views/form.php]: Changing it like discussed above negates the need for the boolean here.

**Review by lonnieezell — CHANGES_REQUESTED** [review]
Good catch! And thanks for the PR.  I think we can clean it up a touch, though.

**Review by lonnieezell — APPROVED** [review]

File: app/Entities/User.php [hunk 3]
```diff
@@ -14,12 +14,14 @@ class User extends ShieldUser
      * @return string
      */
     public function renderAvatar(int $size=52)
-    {
+    {
```
Comment (MGatner) [inline app/Entities/User.php:17]: This makes me think the style Fixer was not run.

**Pull request merged**